Sasquatch/SasquatchSwift/AppDelegate.swift

```diff
@@ -333,8 +333,6 @@ extension AppDelegate: DistributeDelegate {
       // Show alert saying that the app is closing
       self.window?.rootViewController?.present(alertController, animated: true)
     }
-
-    sleep(7)
```

Comment:
The alert should be removed as well. It's useless if we don't block the thread with a delay, because the app is going to be closed right after this callback invocation.